src/shared/lib/calculation/qValuesCalculation.ts

```diff
@@ -0,0 +1,12 @@
+export function calculateQValues(sortedPValues: number[]): number[] {
```

Comment:
I think the file name should be qValueCalculator.ts, or maybe it should specify what kind of qValue calculation it is, such as BenjaminiHochbergFDRCalculator.ts

Reply:
Thanks, Ersin! Yes, Maybe I use "BenjaminiHochbergFDRCalculator.ts" is much better.